src/main/java/com/solutions/shop/model/Customer.java

```diff
@@ -24,15 +25,24 @@ public class Customer {
     private User user;
 
     @Column(name = "first_name")
+    @NotBlank
     private String firstName;
 
     @Column(name = "last_name")
+    @NotBlank
     private String lastName;
 
     @Column(name = "address")
+    @NotBlank
     private String address;
 
-    public Customer(Integer customerId) {
+    public Customer(){
+    }
+
+    public Customer(Integer customerId, String firstName, String lastName, String address) {
```

Comment:
What do you think about starting using lombok? You could use here @RequiredArgsConstructor annotation